src/Uno.Extensions.Reactive.UI/Presentation/Bindings/Collections/Data/DataLayerUpdate.cs

```diff
@@ -58,16 +58,19 @@ public void Complete()
 		{
 			// This will be invoked only when this data layer is a root layer
 
-			// If '_from == null' (i.e. we are initializing the holder) we force the silently to avoid useless events 
-			// as nobody should be listening to the events yet.
-			
-			((ICompositeCallback)this).Invoke(CallbackPhase.All, silently: _context.Type == VisitorType.InitializeCollection);
+			// If this update has been created from a background thread (init),
+			// it might happen that we already have some event handlers, 
+			// so we must make sure to raise event properly (i.e. silently: false)
```

Comment:
```suggestion
			// If this update has been created from a background thread (init),
			// it might happen that we already have some event handlers, 
			// so we must make sure to raise the event properly (i.e. silently: false)
```